extensions/grpc/runtime/pom.xml

```diff
@@ -17,6 +17,10 @@
             <groupId>jakarta.annotation</groupId>
             <artifactId>jakarta.annotation-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>jsr305</artifactId>
```

Comment:
Didn't we remove it?

Reply:
turns out it's needed after all :( The newly added integration test would fail if it was missing. I should actually investigate why it's needed if we have e.g. infinispan-client  together with grpc and doesn't if we don't. I will create a separate issue for it.

Reply:
created: https://github.com/quarkusio/quarkus/issues/12600

Reply:
@cescoffier here's a corresponding JIRA: https://issues.redhat.com/browse/QUARKUS-460

Reply:
Thanks for the explanation @michalszynkiewicz!